mlir/include/mlir/Dialect/Rock/IR/RockAttrDefs.td

```diff
@@ -192,7 +192,7 @@ def Rock_TransformMapAttr : Rock_Attr<"TransformMap"> {
     );
 
     let assemblyFormat = [{
-        `<` $map `by` ` ` `[` $ops `]` `bounds` `=` ($upperBounds^) : (`[` `]`)? `->` $lowerBounds `>`
+        `<` $map `by` ` ` `[` $ops `]` `bounds` `=` ($upperBounds^) : (`[` `]`)? `->` ($lowerBounds^) : (`[` `]`)? `>`
```

Comment:
Why this change ?

Reply:
I found with new partitioning where an empty lower bounds happens. It's like a memref.collapse_shape : memref<1x1x1xf32> into memref<f32>